### ISSUE TITLE
Using pinned peer dep will not work for any version other than pinned

### ISSUE THREAD

**Issue by moltar** [elthrasher/aws-logs-comptroller#5]
Will fail with:

```
npm ERR! Could not resolve dependency:
npm ERR! peer aws-cdk-lib@"2.46.0" from aws-logs-comptroller@0.1.4
npm ERR! node_modules/aws-logs-comptroller
npm ERR!   aws-logs-comptroller@"*" from the root project
```

https://github.com/elthrasher/aws-logs-comptroller/blob/6c4354f342fabcbc9961e95366bddf722221ee9d/package.json#L62

Probably need to have a caret there:

```json
 "aws-cdk-lib": "^2.46.0", 
```

**Comment by elthrasher** [comment]
Hey @moltar apologies for not seeing this before. I wish GitHub would email notifications for new issues! The peer deps issue should be fixed in 0.1.5.

**Comment by moltar** [comment]
> issue should be fixed in 0.1.5.

Nice thank you, sir!

> I wish GitHub would email notifications for new issues

LOL it sure does! 😁 Maybe your config is wrong or they are going to spam?
